waku/v2/protocol/waku_relay.nim

```diff
@@ -23,7 +25,7 @@ const
 
 type
   WakuRelay* = ref object of GossipSub
-    defaultTopics*: seq[string] # Default configured PubSub topics
+    defaultTopics*: seq[PubsubTopic] # Default configured PubSub topics
```

Comment:
i liked that you were enforcing "topic" to be "pubsubtopic" when applicable. Does it make sense to enforce here `defaultPubsubTopic`? When having also contenttopics this seems useful to me.

Reply:
I don't want to add more cognitive load to this review, it already has 30 files.

Same, feel free to do it yourself if you have the bandwidth.